ReactAndroid/src/main/java/com/facebook/react/ReactActivityDelegate.java

```diff
@@ -29,6 +30,10 @@
  * class doesn't implement {@link ReactApplication}.
  */
 public class ReactActivityDelegate {
+
+  private final int REQUEST_OVERLAY_CODE = 1111;
```

Comment:
> because then it is "random" for a developer when that clashes

The developer is not going look into the library to see what code is used. This code is not part  of the app. We can never prevent clash, but this number is too easy to clash with third party lib/user's code.

@SandroMachado that example is a sample app, I don't think libraries can follow the same convention as an app.